mock-identity-system/src/main/java/io/mosip/esignet/mock/identitysystem/service/impl/AuthenticationServiceImpl.java

```diff
@@ -523,9 +550,16 @@ private Map<String,Object> getVerificationDetail(String individualId, JsonNode r
         Iterator<Map.Entry<String, JsonNode>> it = requestedVerifiedClaims.fields();
         while (it.hasNext()) {
             Map.Entry<String, JsonNode> entry = it.next();
-            Optional<List<VerifiedClaim>> result = verifiedClaimRepository.findByIndividualIdAndClaimAndIsActive(individualId, entry.getKey(), true);
+            String trustFramework = "";
+            JsonNode trustFrameworkNode  = requestedVerification.get("trust_framework");
+            if (trustFrameworkNode.has("values")) {
+                trustFramework = String.valueOf(trustFrameworkNode.get("values").get(0)).replace("\"", "");
+            } else if (trustFrameworkNode.has("value")) {
+                trustFramework = String.valueOf(trustFrameworkNode.get("value")).replace("\"", "");
+            }
```

Comment:
Instead of dealing with string replace and has key checks, it is better to deserialise the jsonnode to filter object. and do the checks the object. it is more efficient and cleaner. 
Why is the support added only for "0" element in the values? we should support more than 1 element in the "values" filter.